tests/robot-tests/tests/admin_and_public/bau/data_catalogue.robot

```diff
@@ -236,7 +245,8 @@ Removing search
 
 Validate data catalogue page redirect from slug based urls
     environment variable should be set    PUBLIC_URL
-    user navigates to public frontend    %{PUBLIC_URL}/data-catalogue/${PUPIL_ABSENCE_PUBLICATION_SLUG}/2016-17?newDesign=true
+    user navigates to public frontend
+    ...    %{PUBLIC_URL}/data-catalogue/${PUPIL_ABSENCE_PUBLICATION_SLUG}/2016-17?newDesign=true
```

Comment:
Not you, but `newDesign` can be removed now as we've gone live with the new Data catalogue design